PowerPointLabs/PowerPointLabs/SyncLab/Views/SyncPaneWPF.xaml.cs

```diff
@@ -72,6 +74,21 @@ public int FormatCount
             }
         }
 
+        public void UpdateCopyButtonEnabledStatus()
+        {
+            Selection selection = this.GetCurrentSelection();
```

Comment:
Why not use a passed selection of shapes rather then try to get the current selection again?